app/views/admin/cms/collections/_form.html.erb

```diff
@@ -32,6 +34,13 @@
   </div>
 
   <div>
+    <%= f.label :image, "Default Collection Image" %>
+    <%= f.file_field :image %>
+  </div>
+
+  <img src="<%= @collection.image_url %>">
```

Comment:
does this show even if there's no collection image?

from a validations point of view: can i create a brand new `Collection`, so it will have no legacy `image_url` value, and not upload an `image`?

if so, then this will show an empty `<img />`